stdlib/public/SwiftShims/FoundationShims.h

```diff
@@ -21,9 +21,9 @@
 
 //===--- Layout-compatible clones of Foundation structs -------------------===//
 // Ideally we would declare the same names as Foundation does, but
-// swift's module importer is not yet tolerant of the same struct
+// Swift's module importer is not yet tolerant of the same struct
 // coming in from two different Clang modules
-// (rdar://problem/16294674).  Instead, we copy the definitions here
+// (rdar://problem/16294674). Instead, we copy the definitions here
```

Comment:
I see, but I don't think comments are best place for this convention, for me it doesn't feel right to see two sentences in one line, separated by too much space. Thanks for reply!